fedimint-core/src/api.rs

```diff
@@ -544,7 +544,9 @@ where
             },
             false,
             self.all_peers().total(),
-        );
+        )
+        // downloading a config shouldn't take too long
+        .with_request_timeout(Duration::from_secs(5));
```

Comment:
This limits each request, not the total time fetching config might take, right? Is this the intention?

Reply:
Yes. Each single request. Higher level timeouts are for the strategy itself to figure out.